src/robot/libdoc.py

```diff
@@ -121,11 +121,11 @@
 Examples:
 
   python -m robot.libdoc Dialogs list
-  python -m robot.libdoc Selenium2Library list browser
+  python -m robot.libdoc SeleniumLibrary list browser
```

Comment:
Good catch! Unrelated to this PR otherwise but so small change that including it here is fine.